### DISCUSSION TITLE
mbp.hanabank.com

### DISCUSSION THREAD

**Discussion by malgeum89** [List-KR/community#15] [General]
안녕하세요
하나은행 스마트폰뱅킹 앱인 하나원큐를 사용하고 있습니다 그런데 애드가드앱이 mbp.hanabank.com을 자꾸 차단해서 제가 은행앱을 사용할수 없는 상태입니다
이체도 안돼서!! 제사업에 지장이 있는 상태입니다 왜 은행앱을 차단하시나요?! 빨리 조치를 취해주세요!

**Comment by piquark6046** [comment]
AdGuard DNS 서버를 AdGuard 앱에서 사용하시고 계시나요?
만약 그렇다면 임시 방편으로 Cloudflare DNS 서버를 사용해보시겠어요?

**Reply by malgeum89** [reply]
그게뭔가요?어디서 해야하나요? 전 안드로이드 아이폰 모두 사용하고 있는 사용자에요

**Reply by piquark6046** [reply]
Android: AdGuard 보호 탭 -> DNS 보호 -> DNS 서버에 있습니다.
iOS: AdGuard 보호 탭 -> DNS 보호 -> DNS 서버에 있습니다.

**Reply by malgeum89** [reply]
저는 유료사용자가 아니고.. 일단 앱을 깔기만 했는데 안돼요.. 저 보호탭이 어딨는지 안보입니다.......

**Reply by piquark6046** [reply]
<details><summary>스크린샷</summary>

![Screenshot_20230627_191132_AdGuard](https://github.com/List-KR/List-KR/assets/98787049/8da57725-536a-4420-bf82-285547586c73)


</details> 


iOS, Android 공통으로 AdGuard 하단 탭 버튼의 2번째에 있습니다.

iOS의 경우에는 프리미엄을 구입하시지 않으셨다면, iOS에서 AdGuard DNS를 설정하는 방법은 프로파일을 설치하는 것으로 알려져 있는데 제거하는 방법은 iOS 설정 -> `일반` -> `VPN, DNS 및 기기 관리` -> 구성 프로파일에서 AdGuard DNS DoH을 찾으시고 눌러주세요 -> 제거를 누릅니다.

**Reply by malgeum89** [reply]
일단 해보겠습니다!!

**Reply by piquark6046** [reply]
AdGuard CTO으로 부터 답변이 왔습니다.
https://github.com/List-KR/List-KR/discussions/732#discussioncomment-6292291 를 참고해주세요.
